packages/ui/src/styles/globals.css

```
@@ -5,34 +5,38 @@
 @source "../**/*.{ts,tsx}";
 @source "../../../sanity-blocks/src/**/*.{ts,tsx}";
 
+@plugin "@tailwindcss/typography";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)